opencti-platform/opencti-graphql/src/modules/case/case-resolvers.ts

```diff
@@ -20,6 +20,16 @@ const caseResolvers: Resolvers = {
     case: (_, { id }, context) => findById(context, context.user, id),
     cases: (_, args, context) => findAll(context, context.user, args),
   },
+  Case: {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    __resolveType(obj) {
```

Comment:
Should have a security to handle only Case type ?

Reply:
No because we need to be able to call all types of cases